mlir/lib/Dialect/SparseTensor/Transforms/Utils/LoopEmitter.cpp

```diff
@@ -75,6 +75,40 @@ static Value genSliceStride(OpBuilder &builder, Location loc, Value tensor,
   return createOrFoldSliceStrideOp(builder, loc, tensor, toDim(enc, lvl));
 }
 
+static bool isIntOrFPZero(Attribute attr) {
```

Comment:
we have various helpers for isZero now, but they all are slightly different

not now, but in the longer run we should move all these into one general place again